openmp/runtime/src/kmp_lock.cpp

```diff
@@ -2646,10 +2646,14 @@ static void __kmp_set_drdpa_lock_flags(kmp_drdpa_lock_t *lck,
 kmp_backoff_t __kmp_spin_backoff_params = {1, 4096, 100};
 #else
 // Use nanoseconds for other platforms
+#if KMP_ARCH_ARM64
```

Comment:
I think it would look better to keep the arch specific bits together.
```
#if KMP_ARCH_X86 || KMP_ARCH_X86_64
...
#elif KMP_ARCH_ARM64
...
#else
```